content/en/blog/news/lighthouse-goes-ga.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Lighthouse Goes GA!"
+date: 2020-06-08
+draft: false
+description: >
+  Lighthouse is the default webhook handler for Jenkins X
+categories: [blog]
+keywords: [Jenkins X,2020]
+slug: "lighthouse-goes-ga"
+aliases: []
+author: Andrew Bayer
+---
+
+In May of this year, Jenkins X switched from using [Prow](https://github.com/kubernetes/test-infra/tree/master/prow) as its default webhook handler to using [Lighthouse](https://github.com/jenkins-x/lighthouse). This is the result of almost a year of work, and provides some significant improvements for users of Jenkins X. You can find more information on the differences between Prow and Lighthouse [here](https://jenkins-x.io/docs/reference/components/lighthouse/#comparisons-to-prow).
+
+Most notably, Jenkins X now properly supports GitHub Enterprise as well as github.com, with preview support for GitLab and BitBucket Server. Prow only supports github.com, so until now, Jenkins X users have had to jump through hoops or accept significant limitations to their workflows in order to use any other SCM providers. Lighthouse is tested against every provider we list support for, including the preview support for GitLab and BitBucket Server. While some functionality may not behave exactly the same on all providers, the core functionality of [ChatOps](https://jenkins-x.io/docs/build-test-preview/chatops/) should work on all.
```

Comment:
Could you specify what version of GitLab has "preview support"? From what I can tell from their [website](https://about.gitlab.com/install/ce-or-ee/), there's:

- gitlab.com (SaaS; version of GitLab depends on your pricing plan)
- GitLab Community Edition (self-managed/on-premise; open-source and for free)
- GitLab Enterprise Edition (self-managed/on-premise; proprietary and commercial only)

Reply:
It should be any of them, but yeah, will do.